jwala-tomcat/src/main/resources/data/scripts/linux/jvm-service.sh

```diff
@@ -12,115 +12,27 @@
 ### END INIT INFO
 
 # Source LSB function library.
-. /etc/init.d/functions
 
-NAME=`basename "$0"`
+[Unit]
+Description= jwala service file
+After=network.target
 
-TOMCAT_HOME=@TOMCAT_HOME@
-TOMCAT_HOME_BIN=$TOMCAT_HOME/bin
+[Service]
+Type=forking
```

Comment:
Add a PIDFile option as well when using forking.
The CATALINA_PID is already being set so just use the same file.

See the 'forking' description here:
https://www.freedesktop.org/software/systemd/man/systemd.service.html